packages/canon/css/datatable.css

```diff
@@ -0,0 +1,23 @@
+.canon-DataTablePagination {
+  padding-top: var(--canon-space-5);
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.canon-DataTablePagination--left {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.canon-DataTablePagination--right {
+  justify-content: space-between;
+  align-items: center;
+  gap: var(--canon-space-2);
+  display: flex;
+}
+
+.canon-DataTablePagination--select {
```

Comment:
it comes back to my question above but there seems to be duplicated styles between components.css and datatable.css, is that expected?

Reply:
Yes, that's expected. Unfortunately for now every CSS has to be pushed in 3 places. We make CSS close to each components and then we run `yarn build:css` to build the `component.css` and `styles.css` files.